### DISCUSSION TITLE
Can I sort or nest or save playlists?

### DISCUSSION THREAD

**Discussion by SirGeoffreyB** [audacious-media-player/audacious#1446] [Q&A] [answered]
Hello, I am new to Audacious.
I would like to save a lot of playlists, maybe 60 or 70 or so and am wondering if I can save them in a file folder system some way. Perhaps have them nested to help make them easier to scroll through and find things.
Is that possible?
Or to save play lists in file folders directly, such that I could use Windows Explorer to find playlist files & then have Audacious open them when I click on them?

**Comment by radioactiveman** [comment, marked as answer]
To save a playlist, use `Playlist -> Export`. Where you store it is your decision, Audacious doesn't care about that.
For an overview of all playlists you can enable the `Playlist Manager` plugin, also found in the `Playlist` menu.

**Comment by SirGeoffreyB** [comment]
Thank you very much! 

This looks like it will work well for me, but I'll still need to play around with it to more fully figure out how this all works.

One other wrinkle is that I am doing the editing and organizing at my home and will need to take it to another location. I have been doing that with an external USB drive using Audacity to edit and to play back. Audacious will probably save a lot of disk space as I won't have to duplicate a lot of the hymns for each playlist. Audacious will also be able to play the hymns in their compressed format.

Since the hymns & playlists will need to be moved from computer A -> external drive -> computer B, I guess I will make sure to adjust the file structure so they are identical on all 3 devices.

Thanks again for your time and help. It will really help me out a lot!
